packages/xdl/src/Config.ts

```diff
@@ -1,12 +1,10 @@
-/**
- * @flow
- */
+import getenv from 'getenv';
 
 import * as Env from './Env';
 
-let scheme = process.env.XDL_SCHEME || 'https';
-let host = process.env.XDL_HOST || 'exp.host';
-let port: ?number = parseInt(process.env.XDL_PORT, 10) || null;
+let scheme = getenv.string('XDL_SCHEME', 'https');
+let host = getenv.string('XDL_HOST', 'exp.host');
+let port = getenv.int('XDL_PORT', 0) || null;
```

Comment:
```suggestion
let port = getenv.int('XDL_PORT', 0);
```

Reply:
`null` here means "no port specified". I'd use it as a default value like `getenv.int('XDL_PORT', null)`, but `getenv.int` has `number` as a return type and only accepts `number` as a default value. `0` is used as a default value here because `getenv` throws if the env variable is not set when there's no default value.